src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -22,16 +27,36 @@
  * Represents the in-memory model of the address book data.
  */
 public class ModelManager implements Model {
+    public static final String MESSAGE_NO_NEXT_SESSION_FOUND = "Don't worry, no next session for you this week!";
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
 
     private final AddressBook addressBook;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
-
+    private final Session timeNowSession;
+    private final ArrayList<Session> toSortList;
+    private final HashMap<Session, Person> sessionPersonHashMap;
     private ObservableList<TimeSlot> timeSlots;
     private boolean isDayView;
     private boolean isFullView;
 
+    /**
+     * Class implementing Consumer acting as a helper to add Sessions later
+     * than current time in the week to the list to be sorted.
+     */
```

Comment:
imo since you have two helper methods (the other one below) for `getNextSession` it's worth to create a separate file for these (e.g. NextSessionUtil) esp because they're not relevant to the Model